docker/ci/dockerfiles/current/build.al2.opensearch.x64.arm64.dockerfile

```diff
@@ -114,6 +124,17 @@ RUN git clone -b v0.3.27 --single-branch https://github.com/xianyi/OpenBLAS.git
 ENV LD_LIBRARY_PATH="/usr/local/lib:$LD_LIBRARY_PATH"
 RUN pip3 install cmake==3.26.4
 
+# NodeJS Unofficial Builds
+ARG NODE_VERSION_LIST="18.19.0 20.10.0"
+# https://github.com/opensearch-project/opensearch-build/issues/5178
+# https://github.com/actions/runner/issues/2906
+# https://github.com/actions/runner/issues/3475
+# GitHub enforce nodejs 20 official build in runner 2.317.0 of their actions and CentOS7/AL2 would fail due to having older glibc versions
+# Until https://github.com/actions/runner/pull/3128 is merged or AL2 is deprecated (2025/06) this is a quick fix with unofficial builds support glibc 2.17
+# With changes done similar to this PR (https://github.com/opensearch-project/job-scheduler/pull/702) alongside the image here
+# Only linux x64 is supported in unofficial build until https://github.com/nodejs/unofficial-builds/pull/91 is merged
+RUN if [ `uname -m` = "x86_64" ]; then cd / && for node_version in $NODE_VERSION_LIST; do curl -SLO https://unofficial-builds.nodejs.org/download/release/v$node_version/node-v$node_version-linux-x64-glibc-217.tar.xz; done; fi
```

Comment:
So we will remove this once the `FROM amazonlinux:2` has the right supported versions?

Reply:
1. It seems like very hard to get a right supported version since the original PR on runner binary hasnt any progress.
2. We will remove this or deprecate the entire docker file once AL2 is deprecated by 2025/06.

Thanks.